nifi-nar-bundles/nifi-riemann-bundle/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+      Licensed to the Apache Software Foundation (ASF) under one or more
+      contributor license agreements.  See the NOTICE file distributed with
+      this work for additional information regarding copyright ownership.
+      The ASF licenses this file to You under the Apache License, Version 2.0
+      (the "License"); you may not use this file except in compliance with
+      the License.  You may obtain a copy of the License at
+          http://www.apache.org/licenses/LICENSE-2.0
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+    -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>nifi-nar-bundles</artifactId>
+    <groupId>org.apache.nifi</groupId>
+    <version>0.3.1-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>nifi-riemann-bundle</artifactId>
+  <packaging>pom</packaging>
+  <modules>
+    <module>nifi-riemann-processors</module>
+    <module>nifi-riemann-nar</module>
+  </modules>
+  <repositories>
+    <repository>
+      <id>clojars.org</id>
+      <url>http://clojars.org/repo</url>
+    </repository>
+  </repositories>
+  <dependencies>
+    <dependency>
+      <groupId>com.aphyr</groupId>
+      <artifactId>riemann-java-client</artifactId>
```

Comment:
Ricky,

Here is a flow chart for properly handling licensing.

1) Check license of dependency.  If it is clear what you should do go to step 2.
2) You are done.  But first, was it easy?  If yes go back to step 1 until it is painful.  Once painful go to step 3.
3) Done.

So first question is can we find this dependency without adding a special maven repo to default maven configs?  If not then it has problematic ripple effect of telling folks to pull from additional repos.  

Second question, once you dig into that riemann client codebase you see this pom https://github.com/aphyr/riemann-java-client/blob/master/riemann-java-client/pom.xml.  That shows us there are several additional compile time deps which means transitively we'll have those too.  So you need to validate their licenses and their transitive deps licenses.  If needed will need to update license/notice/etc..

Thanks
Joe